src/edu/csus/ecs/pc2/core/Utilities.java

```diff
@@ -616,8 +618,29 @@ public static String getReportFilename(IReport selectedReport) {
      * @throws FileNotFoundException
      */
     public static String createReport(IReport report, IInternalContest contest, IInternalController controller, boolean printHeaderAndFooter) throws FileNotFoundException {
```

Comment:
The Javadoc header for this method is incomplete (it's missing a description of the `printHeaderAndFooter` parameter).